lib/pagerduty.rb

```diff
@@ -52,13 +51,13 @@ def initialize(service_key)
   #
   # @return [PagerdutyIncident] The triggered incident.
   #
-  # @raise [PagerdutyException] If PagerDuty responds with a status that is not
+  # @fail [PagerdutyException] If PagerDuty responds with a status that is not
```

Comment:
AFAIK `@fail` is not a yardoc tag. http://www.rubydoc.info/gems/yard/file/docs/Tags.md#Tag_List